metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/DataSourceManager.java

```diff
@@ -108,6 +123,47 @@ private synchronized void createDataSource(final String catalogName, final Map p
     }
 
     /**
+     * init properties.
+     *
+     * @throws Exception exception
+     */
+    @PostConstruct
+    public void init() throws Exception {
```

Comment:
This is being also used in hive, tag service, lookup service. Let's talk about this.